data/giants.js

```diff
@@ -0,0 +1,128 @@
+export default [
```

Comment:
Alternatively, you could've created a .json file.

Reply:
json fil created and push to development. Just need to implement it to the code where the data is used.